apps/authz/kinds/rolebinding.cue

```diff
@@ -0,0 +1,43 @@
+package kinds
+
+import (
+	"github.com/grafana/grafana/apps/authz/kinds/v0alpha1"
+)
+
+rolebinding: {
+	kind:       "RoleBinding"
+	pluralName: "RoleBindings"
+	current:    "v0alpha1"
+
+	codegen: {
+		ts: { enabled: false }
+		go: { enabled: true }
+	}
+
+	versions: {
+		"v0alpha1": {
+			schema: {
+				spec:   v0alpha1.RoleBindingSpec
+			}
+		}
+	}
+}
+
+clusterrolebinding: {
```

Comment:
I assume here also, we would change this from cluster -> global.

this means we would have a `GlobalRoleBinding`